buildenv/jenkins/common/pipeline-functions

```diff
@@ -364,6 +364,11 @@ def workflow(SDK_VERSION, SPEC, SHAS, OPENJDK_REPO, OPENJDK_BRANCH, OPENJ9_REPO,
         echo "Using VENDOR_TEST_REPOS = ${VENDOR_TEST_REPOS}, VENDOR_TEST_BRANCHES = ${VENDOR_TEST_BRANCHES}, VENDOR_TEST_SHAS = ${VENDOR_TEST_SHAS}, VENDOR_TEST_DIRS = ${VENDOR_TEST_DIRS}, BUILD_LIST = ${BUILD_LIST}" 
 
         for (name in TARGET_NAMES) {
+            // Checking to see if the test should be excluded
+            if (EXCLUDED_TESTS.contains(name)){
```

Comment:
EXCLUDED_TESTS is a string not an array.

Reply:
I added a tokenize to EXCLUDED_TESTS in the variable-functions, so it is now be an array

Reply:
I changed the excluded_tests in the variable file to be a list instead of tokenizing the string.